src/lib/reif.pl

```diff
@@ -98,3 +98,34 @@
 tmember_t(_P_2, [], false).
 tmember_t(P_2, [X|Xs], T) :-
    if_( call(P_2, X), T = true, tmember_t(P_2, Xs, T) ).
+
+%% if_(+If_1, +Then_0, +Else_0)
+%
+%  A monotonic if-then-else. When the last argument of the predicate If
+%  unifies with true, Then is executed, if the last argument
+%  unifies with false, Else is executed. 
+
+%% =(?X, ?Y, ?T)
+%
+%  Unifies T with false if X and Y are different terms. Otherwise, T is
```

Comment:
This statement is an example of command oriented wording that does not help at all. It claims that "Unifies T with false if X and Y are different terms. Otherwise ...". But:

```
?- =(X,Y,T).
   X = Y, T = true
;  T = false, dif:dif(X,Y).
```
If you are using operationalizing language, this all gets more confusing than it helps.

Reply:
should i directly mention dif(X,Y) here? what would be an appropriate correction?

Reply:
My comment applies to all of your edits. Not just (=)/3. To be able to use `library(reif)` you have to read the paper first. It does not make sense to use (=)/3 alone. Also note that its definition is shorter than your description which is an indication that there is a mismatch. Replacing it with operationalizing/command oriented language is no help.  Just compare it to `library(dcgs)`, which first refers to a tutorial and then keeps documentation quite terse.

Reply:
make sense. I will try a rewrite.